app/src/views/home/maximize_screen.tsx

```diff
@@ -27,12 +25,12 @@ export default function MaximizeScreen({
   route,
   firestoreCtrl,
 }: {
-  readonly user: DBUser;
+  readonly user: any;
   readonly navigation: any;
   readonly route: any;
-  readonly firestoreCtrl: FirestoreCtrl;
+  readonly firestoreCtrl: any;
 }) {
-  const challenge: DBChallenge = route.params?.challenge;
+  const challenge = route.params?.challenge;
```

Comment:
Why did you removed the type there ?

Reply:
I didn't do it on purpose I'll correct this right away !